### PR TITLE
☕️ Revise "environment_vn" - Phần 4

### LINKED ISSUE #1107: Revise "environment_vn" - Phần 4
Phần này được dịch bởi:
* **lkhphuc**
* **nguyenduydu**

Nếu bạn đã dịch phần này, vui lòng bỏ qua việc revise.

### PR DESCRIPTION
**Pull request by duythanhvn**
Close #1107

### CODE REVIEW

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 1]
```diff
@@ -406,11 +406,11 @@ We sometimes additionally apply some penalty to the parameters, using weight dec
 This means that we largely minimize the loss on the training.
 -->
 
-Giả sử ta muốn ước lượng mối liên hệ phụ thuộc $P(y \mid \mathbf{x})$ khi đã có dữ liệu được gán nhãn $(\mathbf{x}_i, y_i)$.
+Giả sử rằng ta muốn ước lượng mối liên hệ phụ thuộc $P(y \mid \mathbf{x})$ khi đã có dữ liệu được gán nhãn $(\mathbf{x}_i, y_i)$.
 Thật không may, các điểm quan sát $x_i$ được thu thập từ một phân phối *mục tiêu* $q(\mathbf{x})$ thay vì từ phân phối *gốc* $p(\mathbf{x})$.
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/environment_vn.md:410]:
```suggestion
Thật không may, các mẫu quan sát $x_i$ được thu thập từ một phân phối *mục tiêu* $q(\mathbf{x})$ thay vì từ phân phối *gốc* $p(\mathbf{x})$.
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 2]
```diff
@@ -442,10 +442,10 @@ Note however, that we only need samples $\mathbf{x} \sim q(\mathbf{x})$; we do n
...
-Đáng buồn là chúng ta không biết tỉ lệ đó nên trước khi làm được bất cứ thứ gì hữu ích ta phải ước lượng được nó.
-Nhiều phương pháp có sẵn sử dụng cách tiếp cận lý thuyết toán tử màu mè cố tái cân bằng trực tiếp toán tử kỳ vọng, sử dụng nguyên lý chuẩn cực tiểu hay entropy cực đại.
-Lưu ý là với các phương thức này yêu cầu ta lấy mẫu từ cả phân phối "đúng" $p$ (bằng cách sử dụng tập huấn luyện) và phân phối được dùng để tạo ra tập kiểm tra $q$ (việc này hiển nhiên là có thể được).
-Tuy nhiên cũng lưu ý rằng ta chỉ cần mẫu $\mathbf{x} \sim q(\mathbf{x})$; ta không hề cần sử dụng đến nhãn $y \sim q(y)$.
+Đáng buồn là ta không biết tỉ lệ đó, nên trước khi ta có thể làm được bất cứ thứ gì hữu ích thì ta cần phải ước lượng được nó.
+Nhiều phương pháp có sẵn sử dụng cách tiếp cận lý thuyết toán tử màu mè nhằm cố tái cân bằng trực tiếp toán tử kỳ vọng bằng cách sử dụng nguyên lý chuẩn cực tiểu hay entropy cực đại.
+Lưu ý là với những phương thức này yêu cầu ta lấy mẫu từ cả phân phối "đúng" $p$ (bằng cách sử dụng tập huấn luyện) và phân phối được dùng để tạo ra tập kiểm tra $q$ (việc này hiển nhiên là có thể được).
```
Comment (tiepvupsu) [inline chapter_multilayer-perceptrons/environment_vn.md:447]:
```suggestion
Lưu ý rằng những phương thức này yêu cầu ta lấy mẫu từ cả phân phối "đúng" $p$ (bằng cách sử dụng tập huấn luyện) và phân phối được dùng để tạo ra tập kiểm tra $q$ (việc này hiển nhiên là có thể được).
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 3]
```diff
@@ -406,11 +406,11 @@ We sometimes additionally apply some penalty to the parameters, using weight dec
...
+Giả sử rằng ta muốn ước lượng mối liên hệ phụ thuộc $P(y \mid \mathbf{x})$ khi đã có dữ liệu được gán nhãn $(\mathbf{x}_i, y_i)$.
 Thật không may, các điểm quan sát $x_i$ được thu thập từ một phân phối *mục tiêu* $q(\mathbf{x})$ thay vì từ phân phối *gốc* $p(\mathbf{x})$.
-Để có được tiến triển, chúng ta cần suy nghĩ lại xem chính xác việc gì đang diễn ra trong quá trình huấn luyện:
-ta duyệt qua tập dữ liệu huấn luyện với nhãn kèm theo $\{(\mathbf{x}_1, y_1), \ldots, (\mathbf{x}_n, y_n)\} và cập nhật vector trọng số của mô hình sau mỗi minibatch.
-Chúng ta thi thoảng cũng áp dụng thêm một lượng phạt nào đó lên các tham số, bằng cách dùng suy giảm trọng số, dropout hoặc các kĩ thuật liên quan khác.
+Để có được tiến triển, chúng ta cần nhìn lại xem chính xác thì việc gì đang diễn ra trong quá trình huấn luyện:
+ta duyệt qua tập dữ liệu huấn luyện cùng với nhãn kèm theo $\{(\mathbf{x}_1, y_1), \ldots, (\mathbf{x}_n, y_n)\}$ và cập nhật vector trọng số của mô hình sau mỗi minibatch.
+Chúng ta đôi khi cũng áp dụng thêm một lượng phạt nào đó lên các tham số, bằng cách dùng suy giảm trọng số, dropout hoặc các kĩ thuật liên quan khác.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/environment_vn.md:413]:
```suggestion
Đôi khi chúng ta cũng áp dụng thêm một lượng phạt nào đó lên các tham số, bằng cách dùng suy giảm trọng số, dropout hoặc các kĩ thuật liên quan khác.
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 4]
```diff
@@ -458,10 +458,10 @@ Now denote by $z_i$ labels which are 1 for data drawn from $p$ and -1 for data d
 Then the probability in a mixed dataset is given by
 -->
 
-Trong trường hợp này có một cách rất hiệu quả, cho kết quả tốt gần ngang ngửa: hồi quy logistic.
+Trong trường hợp này có một cách tiếp cận rất hiệu quả sẽ cho kết quả tốt gần ngang ngửa, đó là: hồi quy logistic.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/environment_vn.md:461]:
```suggestion
Trong trường hợp này có một cách tiếp cận rất hiệu quả và sẽ cho kết quả tốt gần ngang ngửa, đó là: hồi quy logistic.
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 5]
```diff
@@ -458,10 +458,10 @@ Now denote by $z_i$ labels which are 1 for data drawn from $p$ and -1 for data d
...
 Đấy là tất cả những gì ta cần để tính xấp xỉ tỉ lệ xác suất.
 Chúng ta cho học một bộ phân loại để phân biệt giữa dữ liệu được lấy từ phân phối $p(\mathbf{x})$ và phân phối $q(x)$.
-Nếu không thể phân biệt được giữa hai phân phối thì tức là khả năng các mẫu liên quan đến từ một trong hai phân phối là ngang nhau.
+Nếu không thể phân biệt được giữa hai phân phối thì điều đó có nghĩa là khả năng các mẫu liên quan có đến từ một trong hai phân phối là ngang nhau.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/environment_vn.md:464]:
```suggestion
Nếu không thể phân biệt được giữa hai phân phối thì điều đó có nghĩa là khả năng các mẫu liên quan đến từ một trong hai phân phối là ngang nhau.
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 6]
```diff
@@ -458,10 +458,10 @@ Now denote by $z_i$ labels which are 1 for data drawn from $p$ and -1 for data d
...
 Mặt khác, bất kì mẫu nào mà có thể được phân biệt dễ dàng thì cần được đánh trọng số cao lên hoặc giảm đi tương ứng.
```
Comment (minhduc0711) [inline chapter_multilayer-perceptrons/environment_vn.md:465]:
```suggestion
Mặt khác, bất kì mẫu nào mà có thể được phân biệt dễ dàng thì cần được đánh trọng số tăng lên hoặc giảm đi tương ứng.
```

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 7]
```diff
@@ -406,11 +406,11 @@ We sometimes additionally apply some penalty to the parameters, using weight dec
...
 Thật không may, các điểm quan sát $x_i$ được thu thập từ một phân phối *mục tiêu* $q(\mathbf{x})$ thay vì từ phân phối *gốc* $p(\mathbf{x})$.
-Để có được tiến triển, chúng ta cần suy nghĩ lại xem chính xác việc gì đang diễn ra trong quá trình huấn luyện:
-ta duyệt qua tập dữ liệu huấn luyện với nhãn kèm theo $\{(\mathbf{x}_1, y_1), \ldots, (\mathbf{x}_n, y_n)\} và cập nhật vector trọng số của mô hình sau mỗi minibatch.
-Chúng ta thi thoảng cũng áp dụng thêm một lượng phạt nào đó lên các tham số, bằng cách dùng suy giảm trọng số, dropout hoặc các kĩ thuật liên quan khác.
+Để có được tiến triển, chúng ta cần nhìn lại xem chính xác thì việc gì đang diễn ra trong quá trình huấn luyện:
+ta duyệt qua tập dữ liệu huấn luyện cùng với nhãn kèm theo $\{(\mathbf{x}_1, y_1), \ldots, (\mathbf{x}_n, y_n)\}$ và cập nhật vector trọng số của mô hình sau mỗi minibatch.
+Chúng ta đôi khi cũng áp dụng thêm một lượng phạt nào đó lên các tham số, bằng cách dùng suy giảm trọng số, dropout hoặc các kĩ thuật liên quan khác.
 Điều này nghĩa là ta hầu như chỉ đang giảm thiểu giá trị mất mát trên tập huấn luyện.
 
 $$
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/environment_vn.md:416]: Dịch `some penalty`. https://github.com/aivivn/d2l-vn/issues/1165#issuecomment-612567655

File: chapter_multilayer-perceptrons/environment_vn.md [hunk 8]
```diff
@@ -522,9 +522,9 @@ In these approaches, we use one network, $f$ to distinguish real versus fake dat
 We will discuss this in much more detail later.
 -->
 
-*Mạng Đối Sinh* sử dụng một ý tưởng rất giống với mô tả ở trên để thiết kế một *bộ tạo dữ liệu* có khả năng sinh dữ liệu không thể phân biệt được với các mẫu được lấy từ một tập dữ liệu tham chiếu.
+*Mạng Đối sinh* sử dụng một ý tưởng rất giống với mô tả ở trên để thiết kế một *bộ tạo dữ liệu* có khả năng sinh dữ liệu không thể phân biệt được với các mẫu được lấy từ một tập dữ liệu tham chiếu.
```
Comment (rootonchair) [inline chapter_multilayer-perceptrons/environment_vn.md:525]:
```suggestion
*Mạng Đối sinh* sử dụng một ý tưởng rất giống với mô tả ở trên để thiết kế một *bộ sinh dữ liệu* có khả năng tạo dữ liệu không thể phân biệt được với các mẫu được lấy từ một tập dữ liệu tham chiếu.
```

**Pull request merged**